### DISCUSSION TITLE
MMSeg 1.x 中支持变化检测数据集bandon

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#623] [MMSegmentation]
# MMSeg 1.x 中支持变化检测数据集bandon 

在MMSeg 1.x 中支持变化检测数据集BANDON。
参考资料1：[https://github.com/fitzpchao/BANDON](https://github.com/fitzpchao/BANDON) 
参考资料2：[https://github.com/likyoo/open-cd/blob/main/configs/_base_/datasets/bandon.py](https://github.com/likyoo/open-cd/blob/main/configs/_base_/datasets/bandon.py) 

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Medium

**Points**：40

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：AI-Tianlong

**Claim Time**：2023年10月25日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by AI-Tianlong** [comment]
Claim the mission
